### ISSUE TITLE
CSS is broken after #78 (Load URLs in CSS and style attributes)

### ISSUE THREAD

**Issue by rhysd** [Y2Z/monolith#93]
## Pre-condition

- monolith: 295931041c6fd8fb015d351c4832b3d42fcb48d0
- Rust: 1.39.0
- macOS: 10.14

## Repro

```
git clone https://github.com/Y2Z/monolith.git && cd monolith
cargo run -- 'https://github.com/rhysd' > tmp.html
open tmp.html
```

## Expected behavior

Opened HTML page looks the same as https://github.com/rhysd

## Actual behavior

Page is broken:

![スクリーンショット 2019-12-26 16 55 50](https://user-images.githubusercontent.com/823277/71465680-9e598780-2800-11ea-9a3a-0ab4dd717e2f.png)

I pasted the generated HTML file [here](https://gist.githubusercontent.com/rhysd/86055427772b8dd4c20724f206eae67f/raw/7b06d1956f7fff22f7024be6b23e72681ad2f61c/tmp.html) (size: 3.7MB).

## Description

I bisected this issue and confirmed:

- 35f5e13 does not reproduce
- 919e626 reproduces

So I think this is a regression added by #78. I did not look into the generated HTML file, but it seemed that CSS was not applied properly.

**Comment by snshn** [comment]
Hi @rhysd 

thank you very much for reporting this! What seems to be happening is the `integrity` on those links is now doesn't match their contents (since we alter the CSS by embedding images and fonts into them). I'll make a hotfix right now, please feel free to pull and update monolith on your end once this issue is marked closed.

**Comment by rhysd** [comment]
Thank you for quick fix! I confirmed #94 fixed this issue on my side :+1:
